v2/oauth/reuse-website-login.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Reuse website login for desktop and mobile apps
+hide_title: true
+---
+
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs"
+
+
+# Reuse website login for desktop and mobile apps
```

Comment:
Not sure how this page is helpful. The flow for mobile / desktop is very different compared to web apps from the point of view of which lib to use, how to use that lib etc. Once again, we will have to find example tutorial for mobile / desktop apps and link to those, and then add additional comments explaining to users how that tutorial would change in the context of supertokens.

Reply:
I think we should expand this and provide more concrete guidance on how to achieve this, but the main point of this (i.e.: you are re-using web sign-in for mobile) is correct.